src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpHandler.cs

```diff
@@ -1125,18 +808,11 @@ private void SaveServerCredentialsToCache(Uri uri, uint authScheme, ICredentials
                     Interop.WinHttp.WINHTTP_NO_REFERER,
                     Interop.WinHttp.WINHTTP_DEFAULT_ACCEPT_TYPES,
                     secureConnection ? Interop.WinHttp.WINHTTP_FLAG_SECURE : 0);
-                if (requestHandle.IsInvalid)
-                {
-                    throw new HttpRequestException(
-                        SR.net_http_client_execution_error,
-                        WinHttpException.CreateExceptionUsingLastError());
-                }
-                requestHandle.SetParentHandle(connectHandle);
-                
-                state.RequestHandle = requestHandle;
+                ThrowOnInvalidHandle(state.RequestHandle);
+                state.RequestHandle.SetParentHandle(connectHandle);
 
                 // Set callback function.
-                SetStatusCallback(requestHandle, s_staticCallback);
+                SetStatusCallback(state.RequestHandle, WinHttpRequestCallback.s_StaticCallbackDelegate);
```

Comment:
Nit: internal and public properties should not be named using _ or s_. 
(`SetStatusCallback(state.RequestHandle, WinHttpRequestCallback.StaticCallbackDelegate);` instead)